lib/Dialect/Secret/Conversions/SecretToCGGI/SecretToCGGI.cpp

```diff
@@ -614,4 +614,4 @@ struct CombToCGGI : public impl::CombToCGGIBase<CombToCGGI> {
   }
 };
 
-}  // namespace mlir::heir::comb
+}  // namespace mlir::heir::secret
```

Comment:
the namespace on line 42 is still `comb` - the other SecretToScheme are just `mlir::heir` - so maybe we can keep that convention for consistency?